### PR TITLE
Add Steam and savegame locations for Linux.

### PR DESCRIPTION
**Pull request by jld**
Seems to work with local and cloud saves.

A new non-Ironman game doesn't seem to be detected until there's a manual save, but after that autosaves are picked up; I don't know if that's a Linux-specific problem.

### CODE REVIEW

File: backend/engine/linux_finder.py [hunk 1]
```diff
@@ -1,19 +1,15 @@
...
-] # TODO - correct?
-
+STEAM_DIR = 'Steam'
+SAVE_DIR = 'Paradox Interactive/Stellaris/save games'
 
 def find_steam():
-    for folder in STEAM_DIRS:
-        if os.path.isdir(folder): # TODO - use listdir and tolower for case insensitive
-            return path
+    path = os.path.join(str(XDG_DATA_HOME), STEAM_DIR)
```
Comment (benreid24) [inline backend/engine/linux_finder.py:8]: Is this where Steam typically gets installed on Linux? Never used it on Linux myself

**Comment by benreid24** [comment]
Really appreciate this! If possible do you think you could add an Action to package it? The file to change is here: https://github.com/benreid24/Stellaru/blob/master/.github/workflows/builds.yml
You should be able to just copy the Windows job and change a few commands, like source and the powershell zip thing. If not I can add it this weekend :)

File: backend/engine/linux_finder.py (hunk 1 above)
Comment (TytoDan) [reply backend/engine/linux_finder.py:8]: The default location for Steam is: `~/.local/share/Steam/`

Comment (benreid24) [reply backend/engine/linux_finder.py:8]: Thanks! I added the second location

**Review by benreid24 — APPROVED** [review]
Thanks for the help all!

**Pull request merged**